packages/manager/modules/pci/src/projects/project/storages/object-storage/objects/object/addUser/addUser.routing.js

```diff
@@ -41,6 +41,7 @@ export default /* @ngInject */ ($stateProvider) => {
             projectId,
             containerId,
             container.isHighPerfStorage,
+            container.s3StorageType,
```

Comment:
Suggestion: better to use new added params as last, otherwise it could break other functionality if we are using it at some other place.